build/yaml/crd/nsx.vmware.com_subnetports.yaml

```diff
@@ -48,6 +48,44 @@ spec:
           spec:
             description: SubnetPortSpec defines the desired state of SubnetPort.
             properties:
+              attachmentRef:
```

Comment:
it looks like this change is from crd, not from this PR? is it?

Reply:
I recall this portion of code has been removed in PR: https://github.com/vmware-tanzu/nsx-operator/pull/520/files#diff-1c96cd7788a18229fd17ec928994f98a437dfedf84c0ad7fecb4a345435250f5L51, since we don't have attachment in subnet port but in annotation, do you debase the latest code

Reply:
Thanks, this is because we are still using some old commit of pkg/apis in go.mod, so I upgrade the commit in go.mod as well.

Reply:
One question, how we generated the new commit for pkg/apis? if we merged this patch, will the commit of pkg/apis will be changed again?

Reply:
talk offline, we use go get xx@commitid to generate pseudo-randmon-uid here for pkg/apis